lib/publish.js

```diff
@@ -70,6 +70,7 @@ module.exports = async (pluginConfig, context) => {
   debug('Create a release for git tag %o with commit %o', gitTag, gitHead);
   await got.post(urlJoin(gitlabApiUrl, `/projects/${encodedRepoId}/releases`), {
     ...apiOptions,
+    ...proxy,
```

Comment:
As mentioned in my other comment - I think this will cause issues when `proxy` is `undefined`.